lib/src/process/stream.dart

```diff
@@ -59,6 +59,7 @@ abstract class QueryStream extends Stream<QueryUpdate> {
   }
 
   @override
+  // ignore: invalid_method_override_return_type
```

Comment:
any reason for that ignore? Shouldn't we have `WrappedQueryStream` implement `QueryStream` or something like that?

Reply:
Generic type problems. I'm not entirely sure why it does that.